ports/samd/modsamd.c

```diff
@@ -35,9 +35,14 @@
 extern const mp_obj_type_t samd_flash_type;
 #ifdef MICROPY_HW_QSPIFLASH
 extern const mp_obj_type_t samd_qspiflash_type;
+#define SPIFLASH_TYPE samd_qspiflash_type
 #endif
 #if MICROPY_HW_SPIFLASH
 extern const mp_obj_type_t spiflash_type;
+#define SPIFLASH_TYPE spiflash_type
+#endif
+#if MICROPY_HW_MCUFLASH
```

Comment:
Can this be moved up to line 35, and wrap the `extern const mp_obj_type_t samd_flash_type`?

Reply:
Sure & done